PROTO_tests/tests/dtypes_test.py

```diff
@@ -0,0 +1,188 @@
+import numpy as np
+import pytest
+
+import arkouda as ak
+from arkouda import dtypes
+
+"""
+DtypesTest encapsulates arkouda dtypes module methods
+"""
+
+SUPPORTED_NP_DTYPES = [bool, int, float, str, np.int64, np.float64, np.uint8, np.uint64, np.str_]
```

Comment:
we lost `np.bool_` which was present in the original `test_check_np_dtype`

Reply:
Added